### DISCUSSION TITLE
App doesn't launch

### DISCUSSION THREAD

**Discussion by PowaBonobo** [rmottola/Arctic-Fox#200] [Q&A]
Hi everyone,

this may be a stupid question, but I'm not really good at this stuff, but for nostalgia I installed MacOS X 10.6 on my old MacBook Pro, and tried to find a browser to play around with it, I've found this one and downloaded it, put on a USB stick, installed on my MacBook, but when I try to open it a window appears that says:

"It was not possible to open ArticFox due to a problem. Contact the developer to make sure this software works with this version of MacOS..."
I've already verified that my OS is 64bit and the app is too.

In the "More information" tab I've noticed that there was the "libc++" thing that I've read before downloading.
I've downloaded it too, unzipped and I put it in the Library forlder, but still doesn't work...

Any advice or way to fix this?
Thanks in advance!

Sorry again if it's a stupid question and if some translations aren't right, I'm Italian and my OS is in Italian, so some words may be slightly different in English.

**Comment by slavalisoff** [comment]
I have same issue. [Here's the crash log (Pastebin)](https://pastebin.com/uEBKJv2w)
My OS isn't in English too. I have MacPorts installed, also tried copying libraries from the archive.

I can provide additional information if needed

**Comment by rmottola** [comment]
@slavalisoff  If you have MacPorts installed, it should definitely work, no need to download the libc++ stuff. Is it a current version?
It seems related to TLS emulation. I wonder if it is really the same issue, but only @PowaBonobo 
It could be similar, first thing is to check the library is found (and is the correct one for your architecture, 32bit or 64bit) and then if it contains the needed symbols.

**Comment by rmottola** [comment]
Did you try installing the libc provided? I updatedit. Also check latest 45.2 release. Works on my 64bit MacBook.

**Comment by rmottola** [comment]
@PowaBonobo @slavalisoff  please try latest release with supplied libc
